draft-ietf-acme-acme.md

```diff
@@ -1926,8 +1933,8 @@ The server presents a set of challenges in the authorization object it sends to
 client (as objects in the "challenges" array), and the client responds by
 sending a response object in a POST request to a challenge URL.
 
-This section describes an initial set of challenge types.  Each challenge must
-describe:
+This section describes an initial set of challenge types.  The definition of a
+new type of challenge must describe:
```

Comment:
I think this edit is a little bit confusing. What about something like: "In general ACME challenge type must describe:". I think its a little bit strange for the draft to tell you how you would define a new challenge when you're probably reading the draft to understand the existing challenges, not determine the mechanics for what a new challenge type would require.

Reply:
Changed to "The definition of a challenge type includes:", which seems like it will make sense for both readers of this spec and possible future authors of challenge types.